swan-lake/integration-tutorials/sending-emails-from-a-service.md

```diff
@@ -95,162 +95,120 @@ Follow the instructions given in this section to develop the service.
-    > 
-    > ```ballerina
-    > final http:Client hospitalServicesEP = check new (hospitalServicesBackend);
-    > final http:Client paymentEP = check new (paymentBackend);
-    > ```
+    - Use `ReservationRequest` as a parameter indicating that the resource expects a JSON object corresponding to `ReservationRequest` as the payload. 
 
-5. Define records corresponding to the request payload and response payloads.
+    - Use `http:Created|http:NotFound|http:InternalServerError` as the return type to indicate that the response will be an `http:Created` response when the email is sent successfully to the user or the response will be an `http:NotFound` or `http:InternalServerError` response on error.
 
-    ```ballerina
-    type Patient record {|
-        string name;
-        string dob;
-        string ssn;
-        string address;
-        string phone;
-        string email;
-    |};
-
-    type Doctor record {|
-        string name;
-        string hospital;
-        string category;
-        string availability;
-        decimal fee;
-    |};
-
-    type Appointment record {|
-        int appointmentNumber;
-        Doctor doctor;
-        Patient patient;
-        boolean confirmed;
-        string hospital;
-        string appointmentDate;
-    |};
-
-    type PaymentSettlement record {|
-        int appointmentNumber;
-        Doctor doctor;
-        Patient patient;
-        decimal fee;
-        boolean confirmed;
-        string card_number;
-    |};
-
-    type Payment record {|
-        int appointmentNo;
-        string doctorName;
-        string patient;
-        decimal actualFee;
-        int discount;
-        decimal discounted;
-        string paymentID;
-        string status;
-    |};
-
-    type ReservationRequest record {|
-        record {|
-            *Patient;
-            string cardNo;
-        |} patient;
-        string doctor;
-        string hospital_id;
-        string hospital;
-        string appointment_date;
-    |};
-
-    type ReservationResponse record {|
-        int appointmentNo;
-        string doctorName;
-        string patient;
-        decimal actualFee;
-        int discount;
-        decimal discounted;
-        string paymentID;
-        string status;
-    |};
-    ```
+        ![Define the resource](/learn/images/integration-tutorials/sending-emails-from-a-service/define_a_resource.gif)
+
+        The generated service will be as follows.
+
+        ```ballerina
+        service /healthcare on new http:Listener(8290) {
+            resource function post categories/[string category]/reserve(ReservationRequest payload) 
+                    returns http:Created|http:NotFound|http:InternalServerError {
+                
+            }
+        }
+        ```
+
+5. Define a [configurable variable](https://ballerina.io/learn/by-example/#configurability) for the URL of the backend service.
```

Comment:
Shall we say define  "configurable variables" here but only show one is int the demo?

We can remove the "similarly.." part.